masterlist.yaml

```
@@ -3517,6 +3517,22 @@ plugins:
       - Invent
       - Names
       - Relev
+  - name: 'sawyerbatty.esp'
+    url:
+    - link: 'https://www.nexusmods.com/newvegas/mods/75598/'
+      name: 'SawyerBatty TTW'
+    tag:
+      - Actors.ACBS
+      - Delev
+      - Invent.Add
+      - Invent.Remove
+      - Relev
+    inc:
+    - 'jsawyer.esp'
+    - 'Project Nevada - Rebalance.esp'
```

Minor change, for consistency in order of keys.

```suggestion
    inc:
    - 'jsawyer.esp'
    - 'Project Nevada - Rebalance.esp'
    tag:
      - Actors.ACBS
      - Delev
      - Invent.Add
      - Invent.Remove
      - Relev
```